### PR TITLE
switch build from TravisCI to CircleCI

### PR DESCRIPTION
**Pull request by afeld**

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+version: 2
+jobs:
+  build:
+    docker:
+      - image: starefossen/ruby-node:2-8
+    steps:
+      - checkout
+      - run: ./go ci_test
```
Comment (alain-hoang) [inline .circleci/config.yml]: Does bundler ([more info](https://circleci.com/docs/2.0/language-ruby/)) need to run before running this command?

Comment (afeld) [reply .circleci/config.yml]: `go` takes care of it. Happy to split it out if the clarity would help.

https://github.com/18F/before-you-ship/blob/1c7611d2803407ec2ada9ab619cc6b8e1f39e762/go#L12-L18

Comment (erik-burgess) [reply .circleci/config.yml]: I've never used a CI system before, so I'm afraid I can't provide any quality feedback on this.

Comment (alain-hoang) [reply .circleci/config.yml]: If that's the case, I'm for the less code is better approach  👍

**Comment by afeld** [comment]
Thanks y'all!

**Pull request merged**